api/routes/reviews.js

```diff
@@ -49,4 +49,112 @@ router.get('/:review_id', (req, res) => {
   });
 });
 
+/* Update a review */
+router.patch('/:review_id', (req, res) => {
+  let hostname = req.protocol + '://' + req.headers.host;
+
+  // Map form field names to database field names
+  const fields = {
+    overallRating: '"overallRating"',
+    comments: 'comments',
+    strategy: '"strategy"',
+    luck: '"luck"',
+    playerInteraction: '"playerInteraction"',
+    replayValue: '"replayValue"',
+    complexity: '"complexity"',
+    gfKids: '"gfKids"',
+    gfTeens: '"gfTeens"',
+    gfAdults: '"gfAdults"',
+    gfFamilies: '"gfFamilies"',
+    gf2Player: '"gf2Player"',
+    gfLargeGroups: '"gfLargeGroups"',
+    gfSocialDistancing: '"gfSocialDistancing"'
```

Comment:
Are these extra `" "` intentional? I see it in `"overallRating"` but not `comments` for example.

Reply:
Yeah, so it's because of how the fields are setup in the database. For example, here's the schema for the Review table:
```
CREATE TABLE public."Review" (
    id integer NOT NULL,
    "userID" bigint NOT NULL,
    "gameID" bigint NOT NULL,
    "overallRating" numeric(2,1),
    comments text,
    "strategy" numeric(2,1),
    "luck" numeric(2,1),
    "playerInteraction" numeric(2,1),
    "replayValue" numeric(2,1),
    "complexity" numeric(2, 1),
    "gfKids" boolean,
    "gfTeens" boolean,
    "gfAdults" boolean,
    "gfFamilies" boolean,
    "gf2Player" boolean,
    "gfLargeGroups" boolean,
    "gfSocialDistancing" boolean
);
```

I guess one question for you and @zachwreed is whether those double quotes are necessary in the database schema? I feel like ideally the form/request fields just match the database fields exactly, and then I don't need these objects to map the fields.

Reply:
Extracting these field maps out into `utils/db.js` for now